profile_bluesky/startup/instrument/devices/ad_imm_plugins.py

```diff
@@ -23,16 +23,16 @@
 import struct
 
 
-class IMMnLocal(Device):
-    """
-    local interface to the IMM1, & IMM2 plugins
-    """
+# class IMMnLocal(Device):
+#     """
+#     local interface to the IMM1, & IMM2 plugins
+#     """
 
-    capture = Component(EpicsSignalWithRBV, "Capture", kind="config")
-    file_format = Component(
-        EpicsSignalWithRBV, "NDFileIMM_format", string=True, kind="config"
-    )
-    num_captured = Component(EpicsSignalRO, "NumCaptured_RBV")
+#     capture = Component(EpicsSignalWithRBV, "Capture", kind="config")
+#     file_format = Component(
+#         EpicsSignalWithRBV, "NDFileIMM_format", string=True, kind="config"
+#     )
+#     num_captured = Component(EpicsSignalRO, "NumCaptured_RBV")
 
 
 class IMMoutLocal(Device):
```

Comment:
Since all the IMM-specific PVs are not used by this class, looks as if it could be replaced completely by [`ophyd.areadetector.FilePlugin_V34`](https://github.com/bluesky/ophyd/blob/dd4b3e389a0202ecacce39fc3965d703c616b0d4/ophyd/areadetector/plugins.py#L1095)

Reply:
We switched to `ophyd.areadetector.FilePlugin_V34` as a base class for the Imm plugin but that still needs testing.

Reply:
Source has been changed, commit is pending